client/src/components/Lesson/WordCreator/WordDetails.tsx

```diff
@@ -83,6 +93,15 @@ export class WordDetails extends BindingComponent<WordDetailsProps, WordDetailsS
                     hintText='Antonyms'
                     disabled={this.props.disabled}/>
 
+            <h3>Roots</h3>
+            <WordRootSelector
+                roots={this.props.wordInfo.roots}
+                onChange={this.onRootChange.bind(this)}
+                onNewValueChange={ (val: string) => this.setState({ wordRootNewValue: val }) }
+                newValue={this.state.wordRootNewValue || ''}
```

Comment:
Will this ever become undefined, you set it in the constructor, so `|| ''` is unneeded.